yellow/client/api/sculpt/sculpt_characters_fetch_retrieve.py

```diff
@@ -6,15 +6,21 @@
 from ... import errors
 from ...client import AuthenticatedClient, Client
 from ...types import UNSET, Response
+from ...models.file_format_enum import FileFormatEnum
+from ...models.rig_type_enum import RigTypeEnum
```

Comment:
nit: We should avoid relative imports if possible and define in terms of absolute path from the project (see https://peps.python.org/pep-0008/#imports). Applies here and elsewhere.

Reply:
This is an automatically generated method. I think we can leave it as is for now and perhaps reconfigure codegen later (including the other parts) when updating to a new version of the API.

Reply:
SG!